lib/core/versions/latest/RequestHandler.ts

```diff
@@ -142,7 +142,7 @@ export default class RequestHandler implements IRequestHandler {
    * Handles resolve operation.
    * @param shortOrLongFormDid Can either be:
    *   1. A short-form DID. e.g. 'did:sidetree:abc' or
-   *   2. A long-form DID. e.g. 'did:sidetree:<unique-portion>;initial-values=<encoded-original-did-document>'.
+   *   2. A long-form DID. e.g. 'did:sidetree:<unique-portion>?-<methodName>-initial-state=<encoded-original-did-document>'.
```

Comment:
Nit:
I guess one could make a case that this should be:
```suggestion
   *   2. A long-form DID. e.g. 'did:<method-name>:<unique-portion>?-<method-name>-initial-state=<encoded-original-did-document>'.
```